### ISSUE TITLE
[accessibility : Keyboard & navigation] Actions dropdown is not accessible from keyboard

### ISSUE THREAD

**Issue by atreyeemukhopadhyay** [kimchi-project/ginger#280]
Unable to access the dropdown menu items from keyboard.

![2](https://cloud.githubusercontent.com/assets/13199397/14105313/79f33a08-f5c8-11e5-8337-a597f3eb3173.jpg)
